### PR TITLE
[Svace] fix res leak case

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Fix svace issue,
- fix invalid condition to check the args.
- possible res leak in query close function.
- condition to free strings for test models.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_if/gsttensorif.c [hunk 1]
```diff
@@ -354,19 +354,15 @@ gst_tensor_if_set_property_supplied_value (const GValue * value,
 {
   gint i;
   gboolean is_float = FALSE;
-  const gchar *param;
-  gchar **strv;
-  gint num;
+  const gchar *param = g_value_get_string (value);
+  gchar **strv = g_strsplit_set (param, delimiters, -1);
+  gint num = g_strv_length (strv);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_if/gsttensorif.c:359]: I'm not sure how this patch fixes resource leak.

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_if/gsttensorif.c:359]: leak case when calling nnstreamer_query_close() (not this line)

query element allocates TensorQueryConnection handle and tyies to make connection. If connection is failed, it calls nnstreamer_query_close(). In nnstreamer_query_close(), it should be release TensorQueryConnection handle even if closing socket is failed or protocol is not tcp.

https://github.com/nnstreamer/nnstreamer/blob/fcd180087157e3942049565611fa73e66352e4f3/gst/nnstreamer/tensor_query/tensor_query_common.c#L230

Comment (dongju-chae) [reply gst/nnstreamer/tensor_if/gsttensorif.c:359]: Ah, thanks for the detail explanation. I missed the change of line 361.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM ✔

**Review by again4you — APPROVED** [review]
LGTM!

**Pull request merged**